build/yamls/base/kustomization.yml

```diff
@@ -5,7 +5,9 @@ resources:
 - controller-rbac.yml
 - controller.yml
 - agent-rbac.yml
+- whereabouts-rbac.yml
 - agent.yml
+- whereabouts-secret.yml
```

Comment:
I thought we discussed that there should be a flag for `generate-manifest.sh`, that would take care of applying all the yaml patches (except changes to agent-rbac.yml). Wasn't that the case @jianjuns?

Reply:
Initially I had a flag defined for whereabouts (as part of rbac changes). But, since I got inputs to put them directly on the agent-rbac file, I thought we are doing the same for other changes as well.
Is that ok, If I do it in separate PR right after this one?.